docs/webpack-dev-server.config.js

```diff
@@ -51,14 +51,7 @@ const config = {
   module: {
     //eslint loader
     preLoaders: [
-      {
-        test: /\.(js|jsx)$/,
-        loader: 'eslint-loader',
-        include: [path.resolve(__dirname, '../src')],
-        exclude: [
-          path.resolve(__dirname, '../src/svg-icons'),
-        ],
-      },
+
```

Comment:
I'd say remove it. like how we should remove unused vars. less clutter = less confusion :+1: